mods/stairs/init.lua

```diff
@@ -361,7 +486,7 @@ stairs.register_stair_and_slab(
 
 stairs.register_stair_and_slab(
 	"mossycobble",
-	"default:mossycobble",
+    nil,
```

Comment:
This is an unintentional revert of an earlier commit 36d024a716ad929b4c0243d833619d84600fcaea